sdk/program/src/sysvar/slot_history.rs

```diff
@@ -1,8 +1,52 @@
-//! named accounts for synthesized data accounts for bank state, etc.
+//! Access to slots present over the last epoch.
```

Comment:
```suggestion
//! A bitvector of slots present over the last epoch.
```
I'm not set on my suggested wording, but "access" applies something bigger to me